backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlTableMetadata.java

```diff
@@ -390,7 +390,7 @@ private static SqlTableMetadata setInheritTransaction(
 
   @Override
   public TableMetadata removeInherit() {
-    throw new MolgenisException("removeInherit not yet implemented");
+    throw new MolgenisException("remove tableExtends not yet implemented");
```

Comment:
wrong PR ?

Reply:
that is correct, this could be seperate pull request. This is following a question I got where they got this error and then asked 'what does removeInherit' means.

Reply:
More important: build failed :-)

Reply:
build now fixed (was some artefact that only occurs in case of empty database). Should be good to go now.